t/phase.t

```diff
@@ -0,0 +1,273 @@
+# vim:set ft= ts=4 sw=4 et fdm=marker:
```

Comment:
(although i hindsight, i understand what you mean about testing the functionality of the results with the lua-nginx-module tests and loading in resty.core; ill slim down these tests to just examing the JIT-ability; sorry for the confusion!)